90.md

```diff
@@ -0,0 +1,75 @@
+NIP-90
+======
+
+Partially Signed Bitcoin Transactions
+--------------------------------------------
+`draft` `optional` `author:ronaldstoner` `author:Armin Sabouri` `author:satsie` 
+
+# Abstract
+
+This NIP defines an event type for partially signed bitcoin transactions (PSBT) that can be transferred via the Nostr relay network. `1174` is the event kind representing a Bitcoin transaction in standard PSBT format, regardless if the transaction is partially signed or fully finalized.
+
+A special event with kind `1174` which contains an encoded PSBT. PSBTs used under this spec are version 0 [BIP-0174](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki).
```

Comment:
PSBTs have versioning built in and in the spec it recommends to stop parsing if you dont support that version. Libraries should already have handling for this so I think it is fine to allow all kinds of PSBTs for the NIP

Reply:
Thanks @benthecarman! I've added you as a co-author and incorporated all of your feedback.